README.md

```diff
@@ -76,6 +93,20 @@ and
 [NOTICE](https://github.com/InsightSoftwareConsortium/ITK/blob/master/NOTICE)
 files for details.
 
+Supporting ITK
+--------------
+
+ITK is a fiscally sponsored project of [NumFOCUS](https://numfocus.org/), a non-profit dedicated
+to supporting the open source scientific computing community. If you want to
+support ITK's mission to develop and maintain open-source, reproducible
+scientific image analysis software for education and research, please consider
+making a [donation](<!--https://numfocus.org/donate-to-isc-->) to support our efforts.
```

Comment:
Todo: add link

Reply:
The link will be hidden for now due to the HTML comment, so if you wish this can be merged and I would edit the link and make it active when we have one, since we will also need to edit it in the sponsor badge.

Reply:
:+1: sounds good.

Reply:
Link added in 79f4620.